docs/server.md

```diff
@@ -9,18 +9,18 @@
 
 ## Overview
 
-The primary feature of MMS is the model server. It can be used for many types of inference in production settings. It provides an easy-to-use command line interface and utilizes an industry standard [OpenAPI interface](rest_api.md). It has features for pre-processing and post-processing inputs and outputs for customized applications. Support for models from a wide range of deep learning frameworks is achieved through its [ONNX model](https://onnx.ai) export feature.
+MMS model serving can be used for many types of inference in production settings. It provides an easy-to-use command line interface and utilizes a [RESTful API](rest_api.md) handle state prediction requests. Support for models from a wide range of deep learning frameworks is achieved through its [ONNX model](https://onnx.ai) export feature.
```

Comment:
Expand MMS to "MXNet Model Server" ... We wouldn't need model serving again.

Reply:
ah I agree